etcdadm/etcdadm

```diff
@@ -525,11 +525,7 @@ member_restore_from_local_snapshot() {
 }
 
 member_env_file() {
-  local name
-  local env_file
-  name=$(member_name)
-  env_file=$(config_state_dir)/${name}.env
-  echo "${env_file}"
+  echo ${ETCDADM_MEMBER_ENV_FILE:-$(config_state_dir)/etcd-member.env}
```

Comment:
Adding `Environment="ETCDADM_MEMBER_ENV_FILE=/var/run/coreos/etcdadm/etcd-member.env" accordingly to `etcdadm-reconfigure.service` would do the job of actually writing the env file to the fixed path